core/modules/xrdsvc/StreamBuffer.h

```diff
@@ -88,7 +94,11 @@ class StreamBuffer : public XrdSsiStream::Buffer {
     Ptr _selfKeepAlive; ///< keep this object alive until after Recycle() is called.
     util::InstanceCount _ic{"StreamBuffer"}; ///< Useful as it indicates amount of waiting for czar.
 
-    static std::atomic<size_t> _totalBytes;
+    // Members associated with limiting memory use.
+    static std::atomic<int64_t> _totalBytes; ///< Total bytes currently in use by all StreamBuffer instances.
```

Comment:
Once again. An unsigned integer type like `size_t` is a more suitable type for `_totalBytes` and for `_maxTotalBytes`.

Reply:
I respectfully disagree.